### ISSUE TITLE
Toolbar of markdown editor for field in editor component floats on top

### ISSUE THREAD

**Issue by papandreou** [netlify/netlify-cms#1391]
**- Do you want to request a *feature* or report a *bug*?**

*bug*

**- What is the current behavior?**

An editor component that has a field declared as `widget: 'markdown'` will unexpectedly render its toolbar above dropdown menus from the main markdown editor:

![screen shot 2018-05-29 at 16 28 59](https://user-images.githubusercontent.com/373545/40665652-046ccdde-635e-11e8-863d-d4d4ff8f083a.png)

When scrolling down it will also float above the entire toolbar for the main markdown editor, which looks a bit funky:

![screen shot 2018-05-29 at 16 30 36](https://user-images.githubusercontent.com/373545/40665675-20f7dd90-635e-11e8-9cd8-fd8cb87b8166.png)

**- If the current behavior is a bug, please provide the steps to reproduce.**

1. Alter the example Youtube editor component to use the markdown widget for the `Youtube Video ID` field:

  ```js
CMS.registerEditorComponent({
  // Internal id of the component
  id: "youtube",
  // Visible label
  label: "Youtube",
  // Fields the user need to fill out when adding an instance of the component
  fields: [{
    name: 'id',
    label: 'Youtube Video ID',
    widget: 'markdown'
  }],
  // Pattern to identify a block as being an instance of this component
  pattern: /^youtube (\S+)$/,
  // Function to extract data elements from the regexp match
  fromBlock: function(match) {
    return {
      id: match[1]
    };
  },
  // Function to create a text block from an instance of this component
  toBlock: function(obj) {
    return 'youtube ' + obj.id;
  },
  // Preview output for this component. Can either be a string or a React component
  // (component gives better render performance)
  toPreview: function(obj) {
    return (
      '<img src="http://img.youtube.com/vi/' + obj.id + '/maxresdefault.jpg" alt="Youtube Video"/>'
    );
  }
});
```

2. Create a handful of other dummy editor components (so that the list of available components becomes long enough)
3. Create a new entry in a collection
4. Insert a Youtube component via the `+ᵥ` button
5. Click the `+ᵥ` button again

Observe that the dropdown gets rendered under the markdown toolbar.

6. Decrease the height of the browser window so there isn't room for all the fields.
7. Scroll down while the Youtube widget is still open

Observe that the toolbar of the markdown editor belonging to the youtube component gets rendered above the markdown toolbar of the body.

**- What is the expected behavior?**

That the toolbar of the component's markdown editor gets rendered below (`z-index`-wise) dropdowns and the main markdown editor's toolbar.

**- Please mention your versions where applicable.**

Netlify CMS version: 1.8.4
Browser version: Chrome 66.0.3359.181/OSX

Node.JS version: 9.5.0
Operating System: 5.6.0

**- Please link or paste your `config.yml` below if applicable.**

**Comment by polds** [comment]
I'm running into this as well, seems like it stems from [here](https://github.com/netlify/netlify-cms/blob/ccef446d72f96fa8d8db9bf2d0be5633eff79979/packages/netlify-cms-ui-default/src/Dropdown.js#L40). I've confirmed locally that increasing that value solves this issue. Would a PR to modify that z-index value be accepted?

**Comment by erquhart** [comment]
Closed in #2326.
